### DISCUSSION TITLE
ARM64 supported?

### DISCUSSION THREAD

**Discussion by seanzhz** [JanusGraph/janusgraph#3324] [Q&A]
Oh Hi, I just wanna know whether JanusGraph supports the M1(arm64) framework.

**Comment by vtslab** [comment]
This is a very broad question. Did you try and install some JVM on your M1 system and run JanusGraph with the inmemory backend for a start (conf/janusgraph-inmemory.properties)? What backend do you plan to use?
While I do not advise to use untrusted docker images, the following at least suggests that JanusGraph can work on arm64 systems:
https://hub.docker.com/layers/vitaliyazhuravleva/janusgraph-arm/latest/images/sha256-2c801cfaa02232228e4c4ecd5cc7e64d8ba8e5ac6797bd85282ce19fec9e5257?context=explore
